lib/phil/partners/partner.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Phil.Partners.Partner do
+  @moduledoc """
+
+  """
+
+  use Ecto.Schema
+
+  alias Phil.Shipments.BulkShipment
+
+  @types ~w(municipality)a
+
+  schema "partners" do
```

Comment:
are the ids configured as uuids ?

Reply:
> are the ids configured as uuids ?

No, I figured we could decide if we want to do that as a group. Would be easy to add in.